scripts/stablex/setup_environment.js

```diff
@@ -0,0 +1,40 @@
+const StablecoinConverter = artifacts.require("StablecoinConverter")
+const ERC20Mintable = artifacts.require("ERC20Mintable.sol")
+const { getArgumentsHelper } = require("../script_utilities.js")
+
+module.exports = async function (callback) {
+  try {
+    const arguments = getArgumentsHelper()
+    if (arguments.length > 2) {
+      callback("Error: This script accepts optional arguments - <numAccounts=3> <numTokens=3>")
+    }
+    const numAccounts = arguments[0] || 3
+    const numTokens = arguments[1] || 3
+    const instance = await StablecoinConverter.deployed()
+    const accounts = await web3.eth.getAccounts()
+    console.log(`Beginning environment setup with ${numAccounts} accounts and ${numTokens} tokens`)
+
+    const amount = "300000000000000000000"
+
+    // Create and register tokens (feeToken is already registered)
+    const tokens = [await ERC20Mintable.deployed()]
```

Comment:
Maybe we can here make sure that this is really the fee token:
await ERC20Mintable.at(await StablecoinConverter.feeToken()).deployed()]